### PR TITLE
Add test coverage for algorithms in beamsearch.py #6030

### PR DESCRIPTION
**Pull request by ladykkk**

### CODE REVIEW

**Comment by ladykkk** [comment]
Hi, @MridulS  I keep falling about the stye/format(3.8) test. The reason is hook id: black - files were modified by this hook. I've installed Black and do the check, but the problem is still there. Can you please help me to check it? I am really puzzled by it. Thank you.

**Comment by ladykkk** [comment]
fixes#6030

File: networkx/algorithms/traversal/tests/test_beamsearch.py [hunk 1]
```diff
@@ -25,3 +25,8 @@ def test_wide(self):
         G = nx.cycle_graph(4)
         edges = nx.bfs_beam_edges(G, 0, identity, width=2)
         assert list(edges) == [(0, 3), (0, 1), (3, 2)]
+
+    def test_none(self):
```
Comment (MridulS) [inline networkx/algorithms/traversal/tests/test_beamsearch.py:29]: could you be rename this to something like `test_width_none`

Comment (ladykkk) [reply networkx/algorithms/traversal/tests/test_beamsearch.py]: Have done. Thanks for your advice.

**Comment by ladykkk** [comment]
@MridulS I think I wrongly opened PR of update #6033 Improve test coverage for algorithms in betweenness_subset.py here. Sorry for it and I will open a new PR.

**Comment by ladykkk** [comment]
open a new PR in #6087

**Pull request closed without merging**